cocos/ui/UITextFieldEx.cpp

```diff
@@ -949,7 +949,7 @@ namespace ui {
             if (!secureTextEntry)
                 displayText = &this->getString();
             else if(!this->inputText.empty())
-                displayText = &renderLabel->getString();
+                displayText = (const std::string*) &renderLabel->getString();
```

Comment:
Since c++17, the code could be optimize to use `std::string_view instead std::string*`, I will optimize it after this PR merged.